model/user/user.go

```diff
@@ -336,6 +353,15 @@ func (u *DBUser) RemoveRole(role string) error {
 	return event.LogUserEvent(u.Id, event.UserEventTypeRolesUpdate, before, u.SystemRoles)
 }
 
+func (u *DBUser) IsAdmin() bool {
```

Comment:
I don't love this function since it assumes the naming scheme of the admin role -- it's also not accurate, since this naming scheme is only used for roles added from the project page as opposed to mana. Maybe we could do something similar to https://github.com/evergreen-ci/evergreen/blob/f21f81cc987e426ee8d0fe6095c73594e49b081c/model/user/user.go#L375 that just iterates through u.SystemRoles to check permissions and ignores ResourceType